terraform/modules/alertmanager/alertmanager-service.tf

```diff
@@ -112,6 +112,39 @@ resource "aws_ecs_service" "alertmanager_nlb" {
   }
 }
 
+resource "aws_ecs_service" "alertmanager_alb" {
+  for_each = {
+    for _, subnet in data.aws_subnet.private_subnets :
```

Comment:
What does `_` mean?

Reply:
It's a convention in some languages (maybe not terraform?) to mean "this is a variable I have to set but which I don't care about".  I only want the values from the map, not the keys.